evm-adapters/src/sputnik/cheatcodes/mod.rs

```diff
@@ -66,3 +66,14 @@ ethers::contract::abigen!(
             event log_named_string       (string key, string val)
             ]"#,
 );
+
+ethers::contract::abigen!(
+    ConsoleLog,
+    r#"[
```

Comment:
Good for start, we'll eventually replace this with the generic codegened console sol